app/views/comments/_edit.html.erb

```diff
@@ -150,3 +149,15 @@
 </form>
 
 
+
+<style>
```

Comment:
As these styles are also used in `_form.html.erb` we can write it in `app/assets/stylesheet/comment.css` ?

Reply:
done

Reply:
we can also move this to `app/assets/stylesheet/comment.css`

Reply:
done!! I removed it as it was not needed there!!